.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+test.shardio
```

Comment:
Please add an end-of-line character to this file. Best to configure your text editor to ensure that text files end with an end-of-line character.